karbon/karbon_cluster_registration_service_test.go

```diff
@@ -15,14 +15,15 @@ import (
 	"github.com/nutanix-cloud-native/prism-go-client/internal/testhelpers"
 )
 
+const (
+	NoCategoryMappingErrorMsg = "No category mappings provided"
+	NoUUIDErrorMsg            = "uuid in body is required"
```

Comment:
imo, these should be unexported to start out with e.g. `noCategoryMappingErrorMsg` and `noUUIDErrorMsg`